static/locales/ko.json

```
@@ -1631,13 +1631,15 @@
         "exportSettingsFailed": "설정 내보내기 실패: {{error}}",
         "capturingPortrait": "스탠딩 이미지 캡처 중...",
         "generatingCard": "캐릭터 카드 생성 중...",
+        "idleMotionHint": "저장 시 선택한 모션이 대기 모션으로 설정됩니다",
```

Biome (2.4.13)

[error] The key idleMotionHint was already declared.

(lint/suspicious/noDuplicateObjectKeys)